### DISCUSSION TITLE
Getting LUT colors for an arbitrary value

### DISCUSSION THREAD

**Discussion by JeffPS13** [marcomusy/vedo#954] [Q&A] [answered]
Hi, I have a look up table made using: lut = vedo.build_lut(lut_table) and it's applied to a mesh, successfully. Is there a simple way to get lut to return the color that would be applied to an arbitrary value.  I want to apply the color from the lut to correctly color slices of the mesh that I extract. Thanks

**Comment by marcomusy** [comment]
Hi, can you try
```
print(lut.GetColor(xvalue))
```

**Reply by JeffPS13** [reply]
It looks like GetColor takes two input arguments.

**Reply by marcomusy** [reply]
oh sorry try:
```
print(lut.MapValue(xvalue))
```

**Reply by JeffPS13** [reply]
Thanks for the quick reply Marco. When I use: `print(lut.MapValue(xvalue))` I get: `_0000023457dcbe20_p_void`

**Reply by JeffPS13** [reply, marked as answer]
Thanks, Marco. You pointed me in the right direction. This works. 

```
rgb = [0.0, 0.0, 0.0]

# Get the color at index k 
self.lut.GetColor(k, rgb)

# Print the RGB value
print(f"The color at index k is: RGB({rgb[0]}, {rgb[1]}, {rgb[2]})")``
```
